test/ably/resttoken_test.py

```diff
@@ -179,6 +180,12 @@ def test_toke_details_from_json(self):
             TokenDetails.from_json(token_details_str),
         )
 
+    # Issue #71
+    @dont_vary_protocol
+    def test_request_token_float(self):
+        lifetime = datetime.timedelta(hours=4)
+        self.ably.auth.request_token({'ttl': lifetime.total_seconds() * 1000})
```

Comment:
Yes, but the spec also says you can use idiomatic fields such as Ruby which uses seconds instead of milliseconds by default, see https://docs.ably.io/client-lib-development-guide/features/#idl under "Duration & Time types...".  However, if the default in the library so far is milliseconds, and it appear that way based on the documentation, then please leave as milliseconds.  Please confirm what the rest of the lib does.